website/content/docs/concepts/billing-start-date.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Billing Start Date
+description: >-
+   A description of Vault cluster's billing start date used throughout Vault.
+---
+
+# Billing Start Date
+
+This is the billing start date associated with the reporting cluster. This will be the license start date if not configured.
+
+As of 1.15.13, 1.16.7, 1.17.3, 1.18 and later, the billing start date will automatically rollover to the most recent year. 
```

Comment:
We probably want to confirm release versions here, and maybe:

```suggestion
As of 1.15.13, 1.16.7, 1.17.3, 1.18+, the billing start date is automatically adjusted to the most recent billing year. For more information on automatic yearly billing rollover, refer to: [TODO: link to docs].
```

Reply:
What doc link are you referring to here? This is the main concepts doc